packages/cli/src/constructs/webhook-alert-channel-codegen.ts

```
@@ -0,0 +1,134 @@
+      }
+    }
+
+    context.registerAlertChannel(resource.id, 'webhookAlert')
+  }
+
+  gencode (logicalId: string, resource: WebhookAlertChannelResource, context: Context): void {
+    const { webhookType } = resource.config
+    if (webhookType) {
+      const codegen = this.codegensByWebhookType[webhookType]
+      if (codegen) {
+        return codegen.gencode(logicalId, resource, context)
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)